src/utils/metrics.ts

```diff
@@ -271,6 +277,15 @@ export async function insertOss(
         data.target_id = label.id;
         data.target_name = label.name;
         data.target_type = 'label';
+        if (label.name.startsWith(PRODUCT_AREA_LABEL_PREFIX)) {
+          data.product_area = label.name;
+        }
+        else{
```

Comment:
nit: do we need to format? `else{` looks odd.

Reply:
done